python/ui/setup_project.py

```diff
@@ -573,7 +573,7 @@ def retranslateUi(self, Wizard):
         self.label_2.setText(QtGui.QApplication.translate("Wizard", "Please select either a folder or a zip file.", None, QtGui.QApplication.UnicodeUTF8))
         self.storage_map_page.setTitle(QtGui.QApplication.translate("Wizard", "<p></p><font size=18>&nbsp;Define Storages</font><p></p>", None, QtGui.QApplication.UnicodeUTF8))
         self.storage_map_page.setSubTitle(QtGui.QApplication.translate("Wizard", "&nbsp;", None, QtGui.QApplication.UnicodeUTF8))
-        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html_to_set_up\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
+        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
```

Comment:
Hi @shaynacohen, do you think this would be the right link for this case?

Reply:
Indeed! Your update is perfect.